airbyte-integrations/connectors/source-instagram/pyproject.toml

```diff
@@ -18,7 +18,7 @@ include = "source_instagram"
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
 facebook-business = "==19.0.3"
-airbyte-cdk = "0.80.0"
+airbyte-cdk = "1.3.2"
```

Comment:
This could be internal policy for high-usage connectors to pin specifically. But, what if you `^1` here?

Reply:
Done.